src/HonzaBotner/appsettings.BotDev.json

```diff
@@ -36,7 +36,7 @@
         "RoleToWeightMapping": {}
     },
     "InfoOptions": {
-        "Version": "1.0.1-SNAPSHOT",
+        "VersionSuffix": "-LOCAL",
```

Comment:
It would be superb if that leading `-` would be added in the code if the version suffix contains some value.
Aka this option would have only `LOCAL` etc.